installer/PowerToysSetupCustomActions/CustomAction.cpp

```diff
@@ -1376,6 +1376,18 @@ UINT __stdcall TerminateProcessesCA(MSIHANDLE hInstall)
         L"PowerToys.AlwaysOnTop.exe",
         L"PowerToys.exe",
```

Comment:
I don't know if the list gets ordered somehow. But I suggest to kill the runner after the utilities.

Reply:
Sounds sensible. Will make the change :)  Thank you!